shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -725,6 +732,18 @@ object GlutenConfig {
       .booleanConf
       .createWithDefault(true)
 
+  val COLUMNAR_PARQUET_WRITE_BLOCK_SIZE =
+    buildConf("spark.gluten.sql.columnar.parquet.write.blockSize")
+      .internal()
+      .longConf
+      .createWithDefault(128 * 1024 * 1024)
+
+  val COLUMNAR_PARQUET_WRITE_BLOCK_ROWS =
+    buildConf("spark.gluten.sql.native.parquet.write.blockRows")
+      .internal()
+      .longConf
+      .createWithDefault(100 * 1000 * 1000)
+
```

Comment:
Consider using `ParquetOutputFormat.BLOCK_SIZE`

Reply:
`ParquetOutputFormat.BLOCK_SIZE` can not been propagated by Spark SQLConf, so I add two more configs to help user change at runtime.